README.md

```diff
@@ -370,15 +370,6 @@ Other Style Guides
   - [4.5](#arrays--callback-return) Use return statements in array method callbacks. It's ok to omit the return if the function body consists of a single statement following [8.2](#8.2). eslint: [`array-callback-return`](http://eslint.org/docs/rules/array-callback-return)
 
     ```javascript
-    // good
```

Comment:
in this case, there's no bad "map" example. these are "map, reduce, filter", and should be grouped as such.